app/dashboard/static/css/kci-base-2016.6.css

```diff
@@ -240,6 +240,10 @@ div.dataTables_filter {
     font-size: 88%;
     font-weight: normal;
 }
+.build-env-label {
+    font-size: 88%;
```

Comment:
in this case I think we can keep only one style definition, you've put same config from arch-label using a different name.

Reply:
Fixed.